### PR TITLE
chore(ci): fix docker-publish workflow

### PR DESCRIPTION
**Pull request by denis-ev**
forgot to add the repo before the tag

Signed-off-by: Denis Evers <denis-ev@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/04-docker-publish.yml [hunk 1]
```diff
@@ -30,10 +30,6 @@ jobs:
             {{major}}.{{minor}}
             {{major}}
           tag-sha: false # add git short SHA as Docker tag
-      - name: Get VERSION
-        run: |
-          echo "::set-output name=get_version::$(cat VERSION)"
-        id: get_version
```
Comment (denis-ev) [inline .github/workflows/04-docker-publish.yml:36]: removed this because not used

Comment (voryzen) [reply .github/workflows/04-docker-publish.yml:36]: Then, how is it versioned?

Comment (denis-ev) [reply .github/workflows/04-docker-publish.yml:36]:
```
- name: Docker meta
        id: docker_meta
        uses: crazy-max/ghaction-docker-meta@v1
        with:
          images: vivumlab/vivumlab # list of Docker images to use as base name for tags
          tag-semver: |
            {{version}}
             {{major}}.{{minor}}
             {{major}}
           tag-sha: false # add git short SHA as Docker tag
```

Comment (denis-ev) [reply .github/workflows/04-docker-publish.yml:36]: This is grabbing a tag with the fully version (0.6.2) and would release image tag: `0` / `0.6` / `0.6.2` / `latest`

Comment (voryzen) [reply .github/workflows/04-docker-publish.yml:36]: Cool

**Review by voryzen — APPROVED** [review]

**Pull request merged**